### PR TITLE
feat: add profile picture for users

### LINKED ISSUE #30: add profile pictures to users
Fetch profile pictures when the user has `type === "graasp"` and add a `picture` property to the user.

URLs are formatted as : `<root_url>/pictures/<user_id>/large_<picture_id>.jpeg`.

There are variations in size: `large`, `medium` and `small`. Small seems well suited for the purpose here (50px*50px)

### PR DESCRIPTION
**Pull request by spaenleh**
closes #30

### CODE REVIEW

File: src/components/common/CommentEditor.js [hunk 1]
```diff
@@ -146,6 +150,7 @@ class CommentEditor extends Component {
       isEdited: focused,
       selectedTab: focused ? 'write' : 'preview',
     });
+    // dispatchGetUsers();
```
Comment (spaenleh) [inline src/components/common/CommentEditor.js:153]: Will remove this with an update commit

**Review by spaenleh — COMMENTED** [review]
Just a left over comment

File: src/components/common/CommentEditor.js (hunk 1 above)
Comment (juancarlosfarah) [reply src/components/common/CommentEditor.js:153]: OK

**Review by juancarlosfarah — APPROVED** [review]
👏

**Pull request merged**